src/graphics/program-lib/chunks/fresnelSchlick.frag

```diff
@@ -1,7 +1,22 @@
 // Schlick's approximation
 uniform float material_fresnelFactor; // unused
 
+#ifdef HAS_AREA_LIGHTS
```

Comment:
should this be HAS_AREA_LIGHTS or just AREA_LIGHTS?
(looking at CLEARCOAT there as an example)

Reply:
I was influenced by the naming of the javascript var in the standard shader generator which sets the macro - I can change it